src/integrationtests/java/com/aws/greengrass/integrationtests/ShadowManagerTest.java

```diff
@@ -327,6 +327,10 @@ public GetThingShadowResponse getThingShadow(String thingName, String shadowName
                 .mqttConnected(true)
                 .mockCloud(false)
                 .build());
+
+        // wait for retry policy in IotDataPlaneClientFactory#waitForCryptoKeyServiceProvider to fail
+        TimeUnit.SECONDS.sleep(15L);
```

Comment:
can we trigger off of some event rather than just waiting?

Reply:
looks like can wait for the request to be retried and put back on the queue, just fixed